docs/ios/xcsync.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Xcode Sync (xcsync) for .NET MAUI
+description: [Xcsync is designed to provide .NET developers with improved support for editing Apple specific files]
+author: haritha-mohan, mcumming
+ms.date: [08/05/2024]
+---
+# Xcode Sync (Xcsync)
+
+**This article applies to:** ✔️ .NET Core 9 Preview 7 SDK and later versions
+
+Xcode Sync (Xcsync) is a tool that enables developers to leverage Xcode for managing Apple specific files with .NET projects. The tool generates a temporary Xcode project from a .NET project and synchronizes changes to the Xcode files back to the .NET project.
+
+Supported file types include:
+
+- Asset Catalog
+- Plist
+- Storyboard
+- Xib
+
+The tool has 2 commands: generate and sync. Use generate for tracking your changes from your .NET to Xcode and sync for tracking your changes from Xcode to .NET. You can also add the xcsync tool path to your PATH environment variable to invoke the tool directly as a more seamless dotnet command.
+
+## Synopsis
+
+### xcsync-generate
+
+```dotnetcli
+dotnet build /t:xcsync-generate [-p|--project <PROJECT>]
+    [-tfm|--target-framework-moniker <FRAMEWORK>]
+    [-t|--target <TARGET_DIRECTORY>] [-f|--force [true|false]]
+    [-o|--open [true|false]]
+    [-v|--verbosity <LEVEL>]
+    [-d|--dotnet-path <.NET_SDK_PATH>]
+
+dotnet build /t:xcsync-generate -?|-h|--help
+```
```

Comment:
to make the option specifications easier to digest, i followed the documentation style used for other .net cli commands: https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-build

Reply:
The property names come from here: https://github.com/xamarin/xamarin-macios/pull/20975/files#diff-2e24f1b6f44df7580244d00f106fa1a9e2257536fbca9d7bbdc74571549324a9R2424-R2427 
and need to be specified using `dotnet build /t:xcsync-generate /p:[PropertyName]=[Value]`